hack/run-ci-e2e-test.sh

```diff
@@ -50,7 +50,10 @@ cd $WMCO_ROOT
 oc create -f deploy/namespace.yaml
 # The bool flags in golang does not respect key value pattern. They follow -flag=x pattern.
 # -flag x is allowed for non-boolean flags only(https://golang.org/pkg/flag/)
-$OSDK test local ./test/e2e --debug --up-local --operator-namespace=windows-machine-config-operator --local-operator-flags "--zap-level=debug --zap-encoder=console" --go-test-flags "-v -timeout=60m -node-count=$NODE_COUNT $SKIP_NODE_DELETION -ssh-key-pair=$KEY_PAIR_NAME"
-oc delete -f deploy/namespace.yaml 
+# By default we'll skip deletion of the Windows VM to be deleted in the next step
+$OSDK test local ./test/e2e --debug --up-local --operator-namespace=windows-machine-config-operator --local-operator-flags "--zap-level=debug --zap-encoder=console" --go-test-flags "-run=TestWMCO/create -v -timeout=60m -node-count=$NODE_COUNT -skip-node-deletion -ssh-key-pair=$KEY_PAIR_NAME"
+# This step would ensure that that the operator starts again and deletes the VM we created in the previous step
```

Comment:
`Run the deletion tests while testing operator restart functionality. This will clean up VMs created in the previous step.`